app/controllers/application_controller.rb

```diff
@@ -37,6 +37,10 @@ def set_top_categories
     @top_categories ||= Category.take(4)
   end
 
+  def launch_drone?
+    params[:drone].to_i == 3
+  end
+
```

Comment:
I think we should make this a helper method or put it in the orders helper @NickyBobby